python/cudf/requirements/cuda-11.1/dev_requirements.txt

```
@@ -16,7 +16,7 @@ hypothesis
 mimesis
 mypy==0.782
 nbsphinx
-numba>=0.49.0,!=0.51.0
+numba>=>=0.53.1
```

```suggestion
numba>=0.53.1
```

looks like a typo here